ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py

```diff
@@ -217,3 +217,40 @@ def test_kron(
         a=x[0],
         b=x[1],
     )
+
+
+# matrix_exp
+@handle_test(
+    fn_tree="functional.experimental.matrix_exp",
+    dtype_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("numeric"),
+        min_num_dims=1,
+        max_num_dims=2,
+        min_dim_size=1,
+        max_dim_size=50,
+    ),
+)
+def test_matrix_exp(
+    dtype_x,
+    as_variable,
+    with_out,
+    num_positional_args,
+    native_array,
+    container_flags,
+    instance_method,
+    backend_fw,
+    fn_name,
+):
+    dtype, x = dtype_x
+    helpers.test_function(
```

Comment:
ivy_tests/test_ivy/test_functional/test_experimental/test_core/test_linalg.py:245: in test_matrix_exp
    helpers.test_function(
E   TypeError: test_function() missing 1 required keyword-only argument: 'ground_truth_backend'

Reply:
tensorflow: matrix_exp: Value for attr 'T' of uint8 is not in the list of allowed values: bfloat16, half, float, double, int8, int16, int32, int64

Reply:
similarly: jax: matrix_exp: expected A to be a square matrix